src/main/java/io/gravitee/el/spel/SpelTemplateEngine.java

```diff
@@ -34,7 +34,7 @@
  */
 public class SpelTemplateEngine implements TemplateEngine {
 
-    private static final String EXPRESSION_REGEX = "\\{([^#|T|(])";
+    private static final String EXPRESSION_REGEX = "\\{(\\\\S[^#|T|(])";
```

Comment:
As @leleueri said, due to this regex one UT fails.
I propose this one, that seems to work. 
```suggestion
    private static final String EXPRESSION_REGEX = "\\{([^#|T|(|\\s])";
```

Reply:
@brasseld wdyt ?